Cubical/HITs/Sn/Properties.agda

```diff
@@ -87,8 +98,7 @@ wedgeConSn zero zero {A = A} hlev f g hom = F , (λ _ → refl) , right
                          λ i → hcomp (λ k → λ { (i = i0) → hom k
                                                 ; (i = i1) → hom k })
                                        (g (loop i))
-  helper = transport (sym (PathP≡Path _ _ _))
-                     (isOfHLevelPathP' 1 (hlev base base) _ _ _ _)
+  helper = toPathP (isOfHLevelPathP' 1 (hlev _ _) _ _ _ _)
 
   F : (x y : S¹) → A x y
   F base y = f y
```

Comment:
For the `F (loop i) base` case, can't you use `_∙ ∙ _∙ ∙ _`?

Reply:
Homogeneous composition won't work here since A is dependent

Reply:
It's not so much homogeneous vs heterogeneous composition (you're still using `hcomp`) as that there's an artifical restriction in the type of `_∙∙_∙∙_` when we require the middle path to be homogeneous. We could easily generalize it:
```
doubleComp-faces : (A : I → Type ℓ) {x y : A i0} {z w : A i1}
  (p : x ≡ y) (r : z ≡ w)
  → (i : I) (j : I) → Partial (i ∨ ~ i) (A i)
doubleComp-faces A p r i j (i = i0) = p (~ j)
doubleComp-faces A p r i j (i = i1) = r j

_∙∙_∙∙_ : {A : I → Type ℓ} {w x : A i0} {y z : A i1}
  → w ≡ x → PathP A x y → y ≡ z → PathP A w z
_∙∙_∙∙_ {A = A} p q r i =
  hcomp (doubleComp-faces A p r i) (q i)

doubleCompPath-filler : {A : I → Type ℓ} {x y : A i0} {z w : A i1}
  (p : x ≡ y) (q : PathP A y z) (r : z ≡ w)
  → PathP (λ j → PathP A (p (~ j)) (r j)) q (p ∙∙ q ∙∙ r)
doubleCompPath-filler {A = A} p q r j i =
  hfill (doubleComp-faces A p r i) (inS (q i)) j
```

Reply:
Did you add this @aljungstrom ?

Reply:
I vote for fixing this in a separate PR. Changing this here causes issues in other places in the library.

Reply:
Sure! Can you add it as an issue so that we don't forget about it?